CHANGELOG.rst

```diff
@@ -109,6 +111,8 @@ Fixed
 
   Contributed by Tatsuma Matsuki (@mtatsuma)
 
+* Fix python3 tests on travis to ensure test virtualenvs are actually python3. #4863
```

Comment:
As mentioned in other PR, changes to builds/CI/CD/Makefiles don't need Changelog record as it doesn't change the st2 core platform functionality.

Reply:
So, neither of these entries need to be in the changelog? Or just skip the "Fix python3 ..." one?

Reply:
I dropped both changelog entries.